datasets/simulated.py

```diff
@@ -2,8 +2,58 @@
 from benchopt import safe_import_context
 
 with safe_import_context() as import_ctx:
+    import jax
     import numpy as np
-    from benchmark_utils import oracles
+    import jax.numpy as jnp
+    from functools import partial
+    from benchmark_utils.gen_matrices import gen_matrices
+
+
+def quadratic(inner_var, outer_var, hess_inner, hess_outer, cross,
```

Comment:
Could this be merged with `quadratic.py`? Just keep `quadratic` I would say (rename as simulated to avoid failing tests).